bench-tps/src/bench.rs

```diff
@@ -544,12 +545,23 @@ fn get_nonce_blockhash<T: 'static + BenchTpsClient + Send + Sync + ?Sized>(
     client: Arc<T>,
     nonce_account_pubkey: Pubkey,
 ) -> Hash {
-    let nonce_account = client
-        .get_account(&nonce_account_pubkey)
-        .unwrap_or_else(|error| panic!("{:?}", error));
-    let nonce_data = nonce_utils::data_from_account(&nonce_account)
-        .unwrap_or_else(|error| panic!("{:?}", error));
-    nonce_data.blockhash()
+    loop {
+        match client
+            .get_account_with_commitment(&nonce_account_pubkey, CommitmentConfig::processed())
+        {
+            Ok(nonce_account) => match nonce_utils::data_from_account(&nonce_account) {
+                Ok(nonce_data) => return nonce_data.blockhash(),
+                Err(err) => {
+                    info!("Couldn't get data from durable nonce account: {:?}", err);
+                    sleep(Duration::from_secs(1));
+                }
+            },
+            Err(err) => {
+                info!("Couldn't get durable nonce account: {:?}", err);
+                sleep(Duration::from_secs(1));
```

Comment:
Out of curiosity, how much sleeping is required on a normal run?

Reply:
Honestly, I took this particular sleep time from the code for getting normal blockhash. But my theory is that since we use `CommitmentConfig::processed` in both cases, then the sleep time, I guess, should be *block time* which is as far as I know should be around 800ms.

Reply:
Makes sense!